src/js/hooks/cycleCount/useToCountTab.jsx

```diff
@@ -0,0 +1,276 @@
+import React, { useMemo, useState } from 'react';
+
+import { createColumnHelper } from '@tanstack/react-table';
+import _ from 'lodash';
+import { useSelector } from 'react-redux';
+
+import { CYCLE_COUNT_CANDIDATES } from 'api/urls';
+import { TableCell } from 'components/DataTable';
+import TableHeaderCell from 'components/DataTable/TableHeaderCell';
+import Checkbox from 'components/form-elements/v2/Checkbox';
+import { INVENTORY_ITEM_URL } from 'consts/applicationUrls';
+import CycleCountStatus from 'consts/cycleCountStatus';
+import useSpinner from 'hooks/useSpinner';
+import useTableCheckboxes from 'hooks/useTableCheckboxes';
+import useTableDataV2 from 'hooks/useTableDataV2';
+import useTableSorting from 'hooks/useTableSorting';
+import useTranslate from 'hooks/useTranslate';
+import Badge from 'utils/Badge';
+import exportFileFromAPI from 'utils/file-download-util';
+import { mapStringToList } from 'utils/form-values-utils';
+import StatusIndicator from 'utils/StatusIndicator';
+
+const useToCountTab = ({ filterParams }) => {
+  const columnHelper = createColumnHelper();
+  const translate = useTranslate();
+  const spinner = useSpinner();
+  const [pageSize, setPageSize] = useState(5);
+  const [offset, setOffset] = useState(0);
```

Comment:
couldn't those two states be placed in a separate common hook? I can see it being repeated from the `useAllProductsTab`. I can see it even being used in the `useTableDataV2` with those values with ability to override those (while passing those to the `useTableDataV2`)